### DISCUSSION TITLE
Timestamp

### DISCUSSION THREAD

**Discussion by SzTer** [roflcoopter/viseron#863] [General]
Hi,
Is it possible to add a timestamp burnet into the live view and recording?

**Comment by roflcoopter** [comment]
Hi, there is no simple config option sadly but you could do it using the `video_filters` under `recorder`

**Reply by SzTer** [reply]
Sorted.
In ffmpeg under camera and recorder


      video_filters:
        - drawtext=text='%{localtime}':x=(w-tw):y=h-(2*lh):fontcolor=white:box=1:boxcolor=0x00000000@1

**Reply by roflcoopter** [reply]
Nice, thank you! Will add that to the FFmpeg docs.
